src/mjolnir/graphenhancer.cc

```diff
@@ -831,92 +831,6 @@ bool IsIntersectionInternal(const graph_tile_ptr& start_tile,
   return true;
 }
 
-// Get the Headings for the node.
-void GetHeadings(const graph_tile_ptr& tile, NodeInfo& nodeinfo, uint32_t ntrans) {
```

Comment:
were these functions unused?

Reply:
@mandeepsandhu We used to enhance the turnlanes based on if it was an internal edge or not.  That logic was removed and I guess we forgot to remove the functions.